spec/classes/init_spec.rb

```diff
@@ -142,4 +142,25 @@
       }
     end
   end
+
+  describe 'policytools class parameter' do
+    context 'when false' do
+      let(:params) { { :policytools => false } }
+      it 'includes the selinux class' do
+        expect(subject).to contain_class('selinux')
+      end
+      it 'does not manage package { "policycoreutils-python": }' do
+        expect(subject).not_to contain_package('policycoreutils-python')
+      end
+    end
+    context 'when true' do
+      let(:params) { { :policytools => true } }
+      it 'includes the selinux class' do
+        expect(subject).to contain_class('selinux')
+      end
+      it 'manages package { "policycoreutils-python": }' do
+        expect(subject).to contain_package('policycoreutils-python')
```

Comment:
When matching on the affirmative, we want to be explicit about the attributes and their values.

Here's an example

https://github.com/ghoneycutt/puppet-module-selinux/blob/master/spec/classes/init_spec.rb#L11-L19